src/components/DropZone/DropZone.tsx

```diff
@@ -40,7 +42,7 @@ interface State {
   focused: boolean;
   numFiles: number;
   overlayText?: string;
-  size: string;
+  size: Size;
```

Comment:
Should we rename the prop? 
Consumer should only use it when they want a 'static' size and opt-out from the computed size based on node.